### ISSUE TITLE
CalledProcessError: Command 'pushd /opt/conda/lib/python3.9/site-packages/ &&            patch -p0 < /content/alphafold/docker/openmm.patch &&            popd' returned non-zero exit status 1.

### ISSUE THREAD

**Issue by Leon-HXL** [deepmind/alphafold#760]
18%
18/100 [elapsed: 01:14 remaining: 06:26]
Cloning into 'alphafold'...
remote: Enumerating objects: 849, done.
remote: Counting objects: 100% (54/54), done.
remote: Compressing objects: 100% (29/29), done.
remote: Total 849 (delta 17), reused 35 (delta 17), pack-reused 795
Receiving objects: 100% (849/849), 16.06 MiB | 25.97 MiB/s, done.
Resolving deltas: 100% (500/500), done.
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Collecting absl-py==1.0.0 (from -r ./alphafold/requirements.txt (line 1))
  Downloading absl_py-1.0.0-py3-none-any.whl (126 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 126.7/126.7 kB 2.5 MB/s eta 0:00:00
Collecting biopython==1.79 (from -r ./alphafold/requirements.txt (line 2))
  Downloading biopython-1.79-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.7/2.7 MB 11.1 MB/s eta 0:00:00
Collecting chex==0.0.7 (from -r ./alphafold/requirements.txt (line 3))
  Downloading chex-0.0.7-py3-none-any.whl (52 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 52.5/52.5 kB 6.0 MB/s eta 0:00:00
Collecting dm-haiku==0.0.9 (from -r ./alphafold/requirements.txt (line 4))
  Downloading dm_haiku-0.0.9-py3-none-any.whl (352 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 352.1/352.1 kB 13.8 MB/s eta 0:00:00
Collecting dm-tree==0.1.6 (from -r ./alphafold/requirements.txt (line 5))
  Downloading dm_tree-0.1.6-cp39-cp39-manylinux_2_24_x86_64.whl (94 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 94.7/94.7 kB 11.3 MB/s eta 0:00:00
Collecting docker==5.0.0 (from -r ./alphafold/requirements.txt (line 6))
  Downloading docker-5.0.0-py2.py3-none-any.whl (146 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 147.0/147.0 kB 15.2 MB/s eta 0:00:00
Collecting immutabledict==2.0.0 (from -r ./alphafold/requirements.txt (line 7))
  Downloading immutabledict-2.0.0-py3-none-any.whl (4.0 kB)
Collecting jax==0.3.25 (from -r ./alphafold/requirements.txt (line 8))
  Downloading jax-0.3.25.tar.gz (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 18.3 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting ml-collections==0.1.0 (from -r ./alphafold/requirements.txt (line 9))
  Downloading ml_collections-0.1.0-py3-none-any.whl (88 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 88.7/88.7 kB 7.4 MB/s eta 0:00:00
Collecting numpy==1.21.6 (from -r ./alphafold/requirements.txt (line 10))
  Downloading numpy-1.21.6-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (15.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 15.7/15.7 MB 27.2 MB/s eta 0:00:00
Collecting pandas==1.3.4 (from -r ./alphafold/requirements.txt (line 11))
  Downloading pandas-1.3.4-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.5/11.5 MB 38.6 MB/s eta 0:00:00
Collecting scipy==1.7.3 (from -r ./alphafold/requirements.txt (line 12))
  Downloading scipy-1.7.3-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (39.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 39.8/39.8 MB 11.9 MB/s eta 0:00:00
Collecting tensorflow-cpu==2.11.0 (from -r ./alphafold/requirements.txt (line 13))
  Downloading tensorflow_cpu-2.11.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (221.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 221.5/221.5 MB 5.2 MB/s eta 0:00:00
Requirement already satisfied: six in /opt/conda/lib/python3.9/site-packages (from absl-py==1.0.0->-r ./alphafold/requirements.txt (line 1)) (1.16.0)
Collecting jaxlib>=0.1.37 (from chex==0.0.7->-r ./alphafold/requirements.txt (line 3))
  Downloading jaxlib-0.4.7-cp39-cp39-manylinux2014_x86_64.whl (66.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.1/66.1 MB 9.4 MB/s eta 0:00:00
Requirement already satisfied: toolz>=0.9.0 in /opt/conda/lib/python3.9/site-packages (from chex==0.0.7->-r ./alphafold/requirements.txt (line 3)) (0.12.0)
Collecting jmp>=0.0.2 (from dm-haiku==0.0.9->-r ./alphafold/requirements.txt (line 4))
  Downloading jmp-0.0.4-py3-none-any.whl (18 kB)
Collecting tabulate>=0.8.9 (from dm-haiku==0.0.9->-r ./alphafold/requirements.txt (line 4))
  Downloading tabulate-0.9.0-py3-none-any.whl (35 kB)
Collecting websocket-client>=0.32.0 (from docker==5.0.0->-r ./alphafold/requirements.txt (line 6))
  Downloading websocket_client-1.5.1-py3-none-any.whl (55 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.9/55.9 kB 6.6 MB/s eta 0:00:00
Requirement already satisfied: requests!=2.18.0,>=2.14.2 in /opt/conda/lib/python3.9/site-packages (from docker==5.0.0->-r ./alphafold/requirements.txt (line 6)) (2.29.0)
Collecting opt_einsum (from jax==0.3.25->-r ./alphafold/requirements.txt (line 8))
  Downloading opt_einsum-3.3.0-py3-none-any.whl (65 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.5/65.5 kB 8.3 MB/s eta 0:00:00
Collecting typing_extensions (from jax==0.3.25->-r ./alphafold/requirements.txt (line 8))
  Downloading typing_extensions-4.5.0-py3-none-any.whl (27 kB)
Collecting PyYAML (from ml-collections==0.1.0->-r ./alphafold/requirements.txt (line 9))
  Downloading PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (661 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 661.8/661.8 kB 48.7 MB/s eta 0:00:00
Collecting contextlib2 (from ml-collections==0.1.0->-r ./alphafold/requirements.txt (line 9))
  Downloading contextlib2-21.6.0-py2.py3-none-any.whl (13 kB)
Collecting python-dateutil>=2.7.3 (from pandas==1.3.4->-r ./alphafold/requirements.txt (line 11))
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 25.5 MB/s eta 0:00:00
Collecting pytz>=2017.3 (from pandas==1.3.4->-r ./alphafold/requirements.txt (line 11))
  Downloading pytz-2023.3-py2.py3-none-any.whl (502 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.3/502.3 kB 38.6 MB/s eta 0:00:00
Collecting astunparse>=1.6.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading astunparse-1.6.3-py2.py3-none-any.whl (12 kB)
Collecting flatbuffers>=2.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading flatbuffers-23.3.3-py2.py3-none-any.whl (26 kB)
Collecting gast<=0.4.0,>=0.2.1 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading gast-0.4.0-py3-none-any.whl (9.8 kB)
Collecting google-pasta>=0.1.1 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading google_pasta-0.2.0-py3-none-any.whl (57 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.5/57.5 kB 7.5 MB/s eta 0:00:00
Collecting grpcio<2.0,>=1.24.3 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading grpcio-1.54.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.1/5.1 MB 69.0 MB/s eta 0:00:00
Collecting h5py>=2.9.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading h5py-3.8.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.7/4.7 MB 67.4 MB/s eta 0:00:00
Collecting keras<2.12,>=2.11.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading keras-2.11.0-py2.py3-none-any.whl (1.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.7/1.7 MB 65.1 MB/s eta 0:00:00
Collecting libclang>=13.0.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading libclang-16.0.0-py2.py3-none-manylinux2010_x86_64.whl (22.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 22.9/22.9 MB 47.8 MB/s eta 0:00:00
Requirement already satisfied: packaging in /opt/conda/lib/python3.9/site-packages (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13)) (23.1)
Collecting protobuf<3.20,>=3.9.2 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading protobuf-3.19.6-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 50.5 MB/s eta 0:00:00
Requirement already satisfied: setuptools in /opt/conda/lib/python3.9/site-packages (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13)) (67.7.2)
Collecting tensorboard<2.12,>=2.11 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading tensorboard-2.11.2-py3-none-any.whl (6.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.0/6.0 MB 71.0 MB/s eta 0:00:00
Collecting tensorflow-estimator<2.12,>=2.11.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading tensorflow_estimator-2.11.0-py2.py3-none-any.whl (439 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 439.2/439.2 kB 34.5 MB/s eta 0:00:00
Collecting termcolor>=1.1.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading termcolor-2.3.0-py3-none-any.whl (6.9 kB)
Collecting wrapt>=1.11.0 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading wrapt-1.15.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (78 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.3/78.3 kB 10.4 MB/s eta 0:00:00
Collecting tensorflow-io-gcs-filesystem>=0.23.1 (from tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading tensorflow_io_gcs_filesystem-0.32.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (2.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.4/2.4 MB 71.6 MB/s eta 0:00:00
Requirement already satisfied: wheel<1.0,>=0.23.0 in /opt/conda/lib/python3.9/site-packages (from astunparse>=1.6.0->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13)) (0.40.0)
Collecting ml-dtypes>=0.0.3 (from jaxlib>=0.1.37->chex==0.0.7->-r ./alphafold/requirements.txt (line 3))
  Downloading ml_dtypes-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (191 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 191.1/191.1 kB 22.2 MB/s eta 0:00:00
Requirement already satisfied: charset-normalizer<4,>=2 in /opt/conda/lib/python3.9/site-packages (from requests!=2.18.0,>=2.14.2->docker==5.0.0->-r ./alphafold/requirements.txt (line 6)) (2.0.4)
Requirement already satisfied: idna<4,>=2.5 in /opt/conda/lib/python3.9/site-packages (from requests!=2.18.0,>=2.14.2->docker==5.0.0->-r ./alphafold/requirements.txt (line 6)) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /opt/conda/lib/python3.9/site-packages (from requests!=2.18.0,>=2.14.2->docker==5.0.0->-r ./alphafold/requirements.txt (line 6)) (1.24.3)
Requirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.9/site-packages (from requests!=2.18.0,>=2.14.2->docker==5.0.0->-r ./alphafold/requirements.txt (line 6)) (2022.12.7)
Collecting google-auth<3,>=1.6.3 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading google_auth-2.17.3-py2.py3-none-any.whl (178 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 178.2/178.2 kB 20.4 MB/s eta 0:00:00
Collecting google-auth-oauthlib<0.5,>=0.4.1 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading google_auth_oauthlib-0.4.6-py2.py3-none-any.whl (18 kB)
Collecting markdown>=2.6.8 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading Markdown-3.4.3-py3-none-any.whl (93 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 93.9/93.9 kB 12.0 MB/s eta 0:00:00
Collecting tensorboard-data-server<0.7.0,>=0.6.0 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading tensorboard_data_server-0.6.1-py3-none-manylinux2010_x86_64.whl (4.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.9/4.9 MB 73.4 MB/s eta 0:00:00
Collecting tensorboard-plugin-wit>=1.6.0 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading tensorboard_plugin_wit-1.8.1-py3-none-any.whl (781 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 781.3/781.3 kB 57.5 MB/s eta 0:00:00
Collecting werkzeug>=1.0.1 (from tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading Werkzeug-2.3.3-py3-none-any.whl (242 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 242.3/242.3 kB 26.4 MB/s eta 0:00:00
Collecting cachetools<6.0,>=2.0.0 (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading cachetools-5.3.0-py3-none-any.whl (9.3 kB)
Collecting pyasn1-modules>=0.2.1 (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 181.3/181.3 kB 21.0 MB/s eta 0:00:00
Collecting rsa<5,>=3.1.4 (from google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading rsa-4.9-py3-none-any.whl (34 kB)
Collecting requests-oauthlib>=0.7.0 (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading requests_oauthlib-1.3.1-py2.py3-none-any.whl (23 kB)
Collecting importlib-metadata>=4.4 (from markdown>=2.6.8->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading importlib_metadata-6.6.0-py3-none-any.whl (22 kB)
Collecting MarkupSafe>=2.1.1 (from werkzeug>=1.0.1->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading MarkupSafe-2.1.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
Collecting zipp>=0.5 (from importlib-metadata>=4.4->markdown>=2.6.8->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading zipp-3.15.0-py3-none-any.whl (6.8 kB)
Collecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 83.9/83.9 kB 10.9 MB/s eta 0:00:00
Collecting oauthlib>=3.0.0 (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.12,>=2.11->tensorflow-cpu==2.11.0->-r ./alphafold/requirements.txt (line 13))
  Downloading oauthlib-3.2.2-py3-none-any.whl (151 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 151.7/151.7 kB 20.0 MB/s eta 0:00:00
Building wheels for collected packages: jax
  Building wheel for jax (setup.py) ... done
  Created wheel for jax: filename=jax-0.3.25-py3-none-any.whl size=1308508 sha256=7d917dd6a8cf66b25d3f29fd97c73910bfb7e3874df8847f9e5d4973f9084387
  Stored in directory: /root/.cache/pip/wheels/56/33/0a/e55d9492c869c140d03a363f2ceea57c811f95d4a845cbda3e
Successfully built jax
Installing collected packages: tensorboard-plugin-wit, pytz, libclang, flatbuffers, zipp, wrapt, websocket-client, typing_extensions, termcolor, tensorflow-io-gcs-filesystem, tensorflow-estimator, tensorboard-data-server, tabulate, PyYAML, python-dateutil, pyasn1, protobuf, oauthlib, numpy, MarkupSafe, keras, immutabledict, grpcio, google-pasta, gast, dm-tree, contextlib2, cachetools, astunparse, absl-py, werkzeug, scipy, rsa, requests-oauthlib, pyasn1-modules, pandas, opt_einsum, ml-dtypes, ml-collections, jmp, importlib-metadata, h5py, docker, biopython, markdown, jaxlib, jax, google-auth, dm-haiku, google-auth-oauthlib, chex, tensorboard, tensorflow-cpu
  Attempting uninstall: numpy
    Found existing installation: numpy 1.22.3
    Uninstalling numpy-1.22.3:
      Successfully uninstalled numpy-1.22.3
Successfully installed MarkupSafe-2.1.2 PyYAML-6.0 absl-py-1.0.0 astunparse-1.6.3 biopython-1.79 cachetools-5.3.0 chex-0.0.7 contextlib2-21.6.0 dm-haiku-0.0.9 dm-tree-0.1.6 docker-5.0.0 flatbuffers-23.3.3 gast-0.4.0 google-auth-2.17.3 google-auth-oauthlib-0.4.6 google-pasta-0.2.0 grpcio-1.54.0 h5py-3.8.0 immutabledict-2.0.0 importlib-metadata-6.6.0 jax-0.3.25 jaxlib-0.4.7 jmp-0.0.4 keras-2.11.0 libclang-16.0.0 markdown-3.4.3 ml-collections-0.1.0 ml-dtypes-0.1.0 numpy-1.21.6 oauthlib-3.2.2 opt_einsum-3.3.0 pandas-1.3.4 protobuf-3.19.6 pyasn1-0.5.0 pyasn1-modules-0.3.0 python-dateutil-2.8.2 pytz-2023.3 requests-oauthlib-1.3.1 rsa-4.9 scipy-1.7.3 tabulate-0.9.0 tensorboard-2.11.2 tensorboard-data-server-0.6.1 tensorboard-plugin-wit-1.8.1 tensorflow-cpu-2.11.0 tensorflow-estimator-2.11.0 tensorflow-io-gcs-filesystem-0.32.0 termcolor-2.3.0 typing_extensions-4.5.0 websocket-client-1.5.1 werkzeug-2.3.3 wrapt-1.15.0 zipp-3.15.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Processing ./alphafold
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: alphafold
  Building wheel for alphafold (setup.py) ... done
  Created wheel for alphafold: filename=alphafold-2.3.2-py3-none-any.whl size=240360 sha256=fca897fb49f5f5da5fc3cc204f3133dc5377bba0fcb5e8b9e45927184ba85925
  Stored in directory: /tmp/pip-ephem-wheel-cache-kgl0tfdn/wheels/2a/1e/d2/41ba445e8098f9a908cc3021b681297e1d41c8cb7da556d607
Successfully built alphafold
Installing collected packages: alphafold
Successfully installed alphafold-2.3.2
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Requirement already satisfied: pyopenssl in /opt/conda/lib/python3.9/site-packages (23.1.1)
Requirement already satisfied: cryptography<41,>=38.0.0 in /opt/conda/lib/python3.9/site-packages (from pyopenssl) (39.0.1)
Requirement already satisfied: cffi>=1.12 in /opt/conda/lib/python3.9/site-packages (from cryptography<41,>=38.0.0->pyopenssl) (1.15.0)
Requirement already satisfied: pycparser in /opt/conda/lib/python3.9/site-packages (from cffi>=1.12->cryptography<41,>=38.0.0->pyopenssl) (2.21)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
/opt/conda/lib/python3.9/site-packages /content
/bin/bash: /content/alphafold/docker/openmm.patch: No such file or directory

---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
[<ipython-input-2-bad561ebf96d>](https://localhost:8080/#) in <cell line: 11>()
     23 
     24       # Apply OpenMM patch.
---> 25       get_ipython().run_line_magic('shell', 'pushd /opt/conda/lib/python3.9/site-packages/ &&            patch -p0 < /content/alphafold/docker/openmm.patch &&            popd')
     26 
     27       # Make sure stereo_chemical_props.txt is in all locations where it could be searched for.

2 frames
[/usr/local/lib/python3.10/dist-packages/google/colab/_system_commands.py](https://localhost:8080/#) in check_returncode(self)
    135   def check_returncode(self):
    136     if self.returncode:
--> 137       raise subprocess.CalledProcessError(
    138           returncode=self.returncode, cmd=self.args, output=self.output
    139       )

CalledProcessError: Command 'pushd /opt/conda/lib/python3.9/site-packages/ &&            patch -p0 < /content/alphafold/docker/openmm.patch &&            popd' returned non-zero exit status 1.

**Comment by Leon-HXL** [comment]
This error kept showing up since yesterday

**Comment by Augustin-Zidek** [comment]
It looks like your Dockerfile/Colab is not up to date -- that OpenMM patch should no longer be applied.

See https://github.com/deepmind/alphafold/commit/955737f12cc4b27398fe2369417fc2cc79815eb6#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddc

I.e. remove the line

```
get_ipython().run_line_magic('shell', 'pushd /opt/conda/lib/python3.9/site-packages/ && patch -p0 < /content/alphafold/docker/openmm.patch && popd')
```

**Comment by Leon-HXL** [comment]
> It looks like your Dockerfile/Colab is not up to date -- that OpenMM patch should no longer be applied.
> 
> See [955737f#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddc](https://github.com/deepmind/alphafold/commit/955737f12cc4b27398fe2369417fc2cc79815eb6#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddc)
> 
> I.e. remove the line
> 
> ```
> get_ipython().run_line_magic('shell', 'pushd /opt/conda/lib/python3.9/site-packages/ && patch -p0 < /content/alphafold/docker/openmm.patch && popd')
> ```

Thanks for answering! I'm a biologhist and I have no idea in python or coding. I'm using the online colab version of AF2 currently. Can you or the manager who is maitaining this website help fix this issue? Many thanks!

**Comment by Augustin-Zidek** [comment]
I think they fixed it in http://go/gh/sokrypton/ColabFold/commit/1a3d91516f1ce5a9baa1f8e9991adbc2b0a4d0d1, so most likely if you reload the Colab and rerun, it should work fine.
